src/mpecopt_phase_ii.m

```diff
@@ -403,12 +402,12 @@
 stats.h_total = full(mpec_casadi.h_total_fun(x_k,p0));
 stats.total_outer_iterations = k;
 
-stats.n_biactive = sum(x_k(dims.ind_x1)+x_k(dims.ind_x2) < 2*settings.tol_active );
+stats.n_biactive = sum(x_k(dims.ind_x1)+x_k(dims.ind_x2) < 2*opts.tol_active );
 try
-    stats.lambda_g_opt = full(results_nlp.lam_g);
-    stats.lambda_x_opt = full(results_nlp.lam_x);
-    stats.n_active_ineq = sum(abs(lambda_g_opt(ind_g_ineq))>settings.tol);
-    stats.n_active_box = sum(abs((lambda_x_opt))>settings.tol & lbx_bnlp_k~=ubx_bnlp_k);
+    stats.lambda_g_opt = full(results_nlp.lam_g(dims.map_g));
```

Comment:
unrelated to you changes the multipliers should should be in results to be consistent with reg_homotopy?